airstack-modules/modules/airstack/common/index.ts

```diff
@@ -17,9 +18,9 @@ export const BIG_INT_ZERO = BigInt.fromI32(0);
 
 export const SUBGRAPH_SCHEMA_VERSION = "1.0.0";
 
-export const SUBGRAPH_NAME = "AIRSTACK-SUBGRAPH";
-export const SUBGRAPH_VERSION = "AIRSTACK-SUBGRAPH";
-export const SUBGRAPH_SLUG = "AIRSTACK-SUBGRAPH";
+export const SUBGRAPH_NAME = "ens";
+export const SUBGRAPH_VERSION = "v1";
+export const SUBGRAPH_SLUG = "ens-v1";
```

Comment:
In common file, why do we have `ens` specific values? How this will be used?

Reply:
done